flask_monitoringdashboard/core/exception_logger.py

```diff
@@ -0,0 +1,108 @@
+import inspect
+import traceback
+import hashlib
+
+from types import FrameType, TracebackType
+from typing import Union
+
+from sqlalchemy.orm import Session
+from flask_monitoringdashboard.core.user_exception_logger import ExcInfo, ScopedExceptionLogger
+from flask_monitoringdashboard.database import CodeLine, FunctionDefinition
+from flask_monitoringdashboard.database.exception_info import add_exception_info
+from flask_monitoringdashboard.database.stack_trace_snapshot import add_stack_trace_snapshot, get_stack_trace_by_hash
+from flask_monitoringdashboard.database.exception_stack_line import add_exception_stack_line
+from flask_monitoringdashboard.database.function_definition import add_function_definition
+from flask_monitoringdashboard.database.exception_message import add_exception_message
+from flask_monitoringdashboard.database.exception_type import add_exception_type
+
+
+def hash_helper(s: str):
```

Comment:
the fact that it's a helper function is better denoted by prefixing the name with a `_`. Then we can simply call it `_hash` I guess?